addenda10_test.go

```diff
@@ -78,14 +78,10 @@ func BenchmarkAddenda10Parse(b *testing.B) {
 func testAddenda10ValidRecordType(t testing.TB) {
 	addenda10 := mockAddenda10()
 	addenda10.recordType = "63"
-	if err := addenda10.Validate(); err != nil {
-		if e, ok := err.(*FieldError); ok {
-			if e.FieldName != "recordType" {
-				t.Errorf("%T: %s", err, err)
-			}
-		} else {
-			t.Errorf("%T: %s", err, err)
-		}
+	err := addenda10.Validate()
+	// TODO: are we not expecting any errors here?
```

Comment:
return the fieldError

	if addenda10.recordType != "7" {
		fieldError("recordType", NewErrRecordType(7), addenda10.recordType)
	}